lib/elastomer/client/delete_by_query.rb

```diff
@@ -1,3 +1,5 @@
+require 'active_support/core_ext/hash'
```

Comment:
@look I'm new to Ruby, so please, let me know whether this is how I'm supposed to use `ActiveSupport::Hash`. Thank you!